prod/westeurope/internal/api/functions_app_r3/function_app/terragrunt.hcl

```diff
@@ -86,7 +86,7 @@ inputs = {
   app_service_plan_info = {
     kind     = "elastic"
     sku_tier = "ElasticPremium"
-    sku_size = "EP3"
+    sku_size = "EP1"
   }
 
   runtime_version = "~3"
```

Comment:
Maybe here we can specify the pre_warmed_instance_count like functions_bonusapi_r3
On Azure now the pre_warmed_instance_count is set to 1 for io-p-fn3-app

Reply:
mmmm sorry, I am not sure I get you:

the **pre_warmed_instance_count** is already set to 1 (one) as per default:

```
variable "pre_warmed_instance_count" {
  type = number
  default = 1
}
```

Do you want to reassign the value here in terragrunt to be more explicit?

Reply:
Oh sorry, it's ok omit and use default, my mistake it's because I saw in functions_bonusapi_r3
```
pre_warmed_instance_count = 1
```
Only to use same writing rule.

Reply:
..... most likely weeks ago we used pre_warmed_instance_count=3 (or five) then we move to 1.
So that in some files you'll find the value set to 1 (one) even if you can skip it ... since it's the default.

Reply:
ok, anyway for me it's the PR it's ok you can merge for me.